horreum-backend/src/main/java/io/hyperfoil/tools/horreum/datastore/BackendResolver.java

```diff
@@ -13,11 +13,10 @@ public class BackendResolver {
     @All
     List<Datastore> backendStores;
 
-    public Datastore getBackend(DatastoreType type){
-        return backendStores.stream()
-                .filter( store -> store.type().equals(type))
-                .findFirst()
-                .orElseThrow(() -> new IllegalStateException("Unknown datastore type: " + type));
+    public Datastore getBackend(DatastoreType type) {
+        return backendStores.stream().filter(store -> store.type().equals(type)).findFirst().orElseThrow(
+                () -> new IllegalStateException("Unknown datastore type: " + type)
+        );
```

Comment:
I think we should be careful to preserve readability of code, especially when processing streams.

My personal preference when chaining methods together, i.e. with stream processing, is to split each method call out onto separate lines. 
I know this is a personal preference, but i find it really help when skimming a method to be able to read what a stream processor is doing

Reply:
Yeah I agree, I missed this option (I have the same preference)

Btw this is what I'd like to discuss, trying to find a common agreement on the codestyle

Reply:
Now it should follow what you were suggesting:
```java
    public Datastore getBackend(DatastoreType type) {
        return backendStores.stream()
                .filter(
                        store -> store.type()
                                .equals(type)
                )
                .findFirst()
                .orElseThrow(() -> new IllegalStateException("Unknown datastore type: " + type));
    }
```